### DISCUSSION TITLE
[LionSelectRich] is not assignable to parameter of type 'Constructor<LitElement>'

### DISCUSSION THREAD

**Discussion by giuseppedandria** [ing-bank/lion#1749] [Q&A]
Hello everyone,
I am experiencing problems in using select-rich.

The error in my terminal is as follows:
`Argument of type 'typeof LionSelectRich' is not assignable to parameter of type 'Constructor<LitElement>'.
Types of construct signatures are incompatible.
Type 'new () => LionSelectRich' is not assignable to type 'new (...args: any[]) => LitElement'.
Type 'LionSelectRich' is missing the following properties from type 'LitElement': renderOptions, __childPart, createRenderRoot, update, and 306 more.ts(2345)`

Attached is the code snippet
![Snippet](https://user-images.githubusercontent.com/20876028/180821176-999f4437-3037-499c-9ab8-11a0235468a7.png)

Do you have any suggestions?

**Comment by AdrianaSantorio** [comment]
Got the same error, wasn't able to find a solution, a similar issue arised also with the calendar component.
